.devcontainer/Dockerfile

```diff
@@ -22,17 +26,41 @@ RUN apt update && apt upgrade -y
 # Install Git
 RUN apt install -y git
 
+# Install necessary packages for Gazebo Garden and its dependencies
+RUN apt-get update && apt-get install -y \
+    software-properties-common \
+    wget \
+    lsb-release \
+    gnupg \
+    curl
+
+# Install Gazebo
+RUN wget https://packages.osrfoundation.org/gazebo.gpg -O /usr/share/keyrings/pkgs-osrf-archive-keyring.gpg
+RUN echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/pkgs-osrf-archive-keyring.gpg] http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/gazebo-stable.list > /dev/null
```

Comment:
no need to use `sudo tee`, use only `tee`

Reply:
fine